featuretools/primitives/standard/aggregation_primitives.py

```diff
@@ -248,11 +344,17 @@ def get_function(self):
 
 
 class Skew(AggregationPrimitive):
-    """Computes the skewness of a data set.
+    """Computes the extent to which a distribution differs from a normal distribution.
+
+    Description:
+        For normally distributed data, the skewness should be about 0.
+        A skewness value > 0 means that there is more weight in the
+        left tail of the distribution.
 
-    For normally distributed data, the skewness should be about 0. A skewness
-    value > 0 means that there is more weight in the left tail of the
-    distribution.
+    Examples:
+        >>> skew = Skew()
+        >>> skew([1, 2, 3, 4, 5, None])
```

Comment:
can we update this example to not return 0.0?

Reply:
Fixed.